<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgEditMultipleSongs</class>
 <widget class="QDialog" name="DlgEditMultipleSongs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SkauTan: Edit multiple songs</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Songs:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="lwSongs">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="chbAuthor">
       <property name="text">
        <string>Set author:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leAuthor"/>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="chbTitle">
       <property name="text">
        <string>Set title:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leTitle"/>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="chbGenre">
       <property name="text">
        <string>Set genre:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leGenre"/>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="chbMPM">
       <property name="text">
        <string>Set MPM:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="leMPM"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
